libsession/src/main/java/org/session/libsession/messaging/jobs/InviteContactsJob.kt

```diff
@@ -119,7 +119,7 @@ class InviteContactsJob(val groupSessionId: String, val memberSessionIds: Array<
                     members.set(toSet)
                 }
             }
-            val failures = results.filter { true }
+            val failures = results.filter { !it.success }
             // if this is the final failure, display a message
             if (failures.isNotEmpty() && failureCount + 1 >= maxFailureCount) {
```

Comment:
The second condition `failureCount + 1 >= maxFailureCount` is practically always `true` due to `failureCount` initialised to 0 (and also never increased). If there's no need to change this variable, is it safe to remove it?